requirements.in

```diff
@@ -1,7 +1,7 @@
 --extra-index-url="https://packages.dea.ga.gov.au"
 
 datacube[s3]==1.8.6
-aiobotocore[awscli, boto3]==2.0.1
+aiobotocore[awscli, boto3]==2.1.0
```

Comment:
What do you think about removing `awscli` as a dependency?

We need it to be available in the `PATH`, not necessarily in the same Python environment. Not specifying it could help the library solver.

Reply:
We actually need the AWS CLI. The method for getting data onto S3 is to use the `aws s3 sync` command line action. Perhaps not the simplest idea, but very reliable.

Reply:
Oh I see what you're saying. I think it's better to have it all installed from the aibotocore lib, as that forces the versions, I think... not sure.